lib/src/room.dart

```diff
@@ -1107,4 +1133,10 @@ class KurentoRoom extends ChangeNotifier implements Room {
       _isReconnecting = false;
     }
   }
+
+  void _removeTracksByAgent(int incomingTrackId) {
+    _tracksByAgentId.removeWhere((key, value) {
+      return value.firstWhereOrNull((item) => item == incomingTrackId) != null;
```

Comment:
I guess we may simplify this line by use any():
```suggestion
      return value.any((item) => item == incomingTrackId);
```

Reply:
Ahh, I missed this. how about `contains`? its simpler.

Reply:
yes, if you don't need to have a custom `test` function and this case use `contains` could be even better.